scripts/events/saveEvent.js

```diff
@@ -0,0 +1,12 @@
+import disableEditing from "/scripts/factory/disableEditing.js";
+import toggleButton from "/scripts/factory/toggleButton.js";
+import removeEventListeners from "/scripts/factory/removeEvents.js";
+
+function saveEvent(buttons, inputElement, tasks, id, eventHandlers) {
+  disableEditing(inputElement, false);
+  tasks[id].content = inputElement.textContent;
+  toggleButton(...buttons);
+  removeEventListeners(buttons, ...eventHandlers);
```

Comment:
why it is needed to remove the listeners on task save?

Reply:
@sam-wtag otherwise they are registered multiple times.